pegasus-spark-analyser/src/main/java/com/xiaomi/infra/pegasus/spark/analyser/ColdBackupConfig.java

```diff
@@ -7,6 +7,8 @@
   public String policyName = "every_day";
   // creating time of cold backup data, for example: 2019-09-11
   public String coldBackupTime = "";
+  // different kv store has different restore version, default set "RecordRestoreV1"
```

Comment:
"different kv store" 这个类应该只支持 pegasus, 为什么还关系到其他 kv store?

```suggestion
  // pegasus has different pegasus data versions, default is "DataVersion1"
```